src/tests/api/BavUnhappyPath.test.ts

```diff
@@ -104,7 +106,30 @@ describe("BAV CRI unhappy path tests", () => {
 		});
 	});
 
-	describe("/authorization Endpoint Unhappy Path Tests", () => {
+	describe("BAV CRI: /verify-account Endpoint Partial Match Athena Output Test", () => {
```

Comment:
Can you delete this Unhappy path test if we're already validating this scenario in the HappyPath test file